starknet/src/types/proposal.cairo

```diff
@@ -16,15 +16,26 @@ const TWO_POWER_32: u128 = 0x100000000;
 const TWO_POWER_64: u128 = 0x10000000000000000;
 const TWO_POWER_96: u128 = 0x1000000000000000000000000;
 
+/// Definition of a proposal.
 #[derive(Clone, Drop, Serde, PartialEq)]
 struct Proposal {
+    /// The timestamp at which the voting period starts.
     start_timestamp: u32,
+    /// The timestamp at which a proposal can be early finalized.
+    /// Spaces that do not want this feature should ensure `min_voting_duration`
+    /// and `max_voting_duration` are set at the same value.
     min_end_timestamp: u32,
+    /// The timestamp at which the voting period ends.
     max_end_timestamp: u32,
+    /// The finalization status of the proposal.
     finalization_status: FinalizationStatus,
+    /// The hash of the execution payload.
     execution_payload_hash: felt252,
+    /// The address of the execution strategy.
     execution_strategy: ContractAddress,
+    /// The address of the proposal author.
     author: UserAddress,
+    /// A snapshot of the active voting strategies at the time of proposal creation.
```

Comment:
Bit array where the index of each each bit corresponds to whether the voting strategy at that index is active at the time of proposal creation.

Reply:
495aa612cd6309f4c716b43928fe3e934eac70b0